app/workers/gemini/tasks.py

```
@@ -170,6 +176,71 @@ def _archive_processed_file(file_path: str):
         logger.info(f"📂 File archived: {source_path.name} -> tenders_positions/")
 
 
+def _regenerate_reports_for_lot(tender_id: str, lot_id: str, ai_result: Dict[str, Any]):
+    """
+    Регенерирует MD и chunks файлы для лота с реальными AI данными.
```

Ruff (0.14.2)

Docstring contains ambiguous `с` (CYRILLIC SMALL LETTER ES). Did you mean `c` (LATIN SMALL LETTER C)?

(RUF002)